cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/preview/PreviewHttpHandler.java

```diff
@@ -47,10 +51,12 @@
   private static final Gson GSON = new GsonBuilder().registerTypeAdapter(Schema.class,
                                                                          new SchemaTypeAdapter()).create();
   private final PreviewManager previewManager;
+  private int limit;
 
   @Inject
-  PreviewHttpHandler(PreviewManager previewManager) {
+  PreviewHttpHandler(CConfiguration cConf, PreviewManager previewManager) {
     this.previewManager = previewManager;
+    limit = cConf.getInt("preview.record.limit", 50);
```

Comment:
This is not a permanent solution and it is an alpha feature. so okay not having it in cdap-default